### ISSUE TITLE
Vertical alignment of callout text

### ISSUE THREAD

**Issue by janicecmt** [asciidoctor/asciidoctor#1999]
It appears that the callout numerical icons are automatically vertically aligned as center alignment with the default HTML output, meaning that if the text goes for more than one line the number is not at the start of the text. I think most people would prefer that it be at the start of the relevant text (I know I would). Ideally, the default should be top alignment but at a minimum there could/should be an option that can be set either for specific code blocks or in the document header to control the alignment and set it to the desired value.

<img width="947" alt="calloutverticalalignment" src="https://cloud.githubusercontent.com/assets/24766266/21935121/2df99462-d97a-11e6-929a-af08fd2b2b77.png">

**Comment by mojavelinux** [comment]
I agree, this seems to be the wrong default. (Though keep in mind you can adjust this by modifying the stylesheet).

Here are the styles that seem to fix it and still preserve the alignment for a single line of text.

```css
.colist>table tr>td:first-of-type {
  vertical-align: top;
  padding-top: 0.4em;
}
```
